core/src/solve.jl

```diff
@@ -346,6 +356,30 @@ function formulate!(level_control::LevelControl, p::Parameters)::Nothing
     return nothing
 end
 
+function formulate!(flow_boundary::FlowBoundary, p::Parameters, u)::Nothing
+    (; connectivity, basin) = p
+    (; graph, flow) = connectivity
+    (; node_id, flow_rate) = flow_boundary
+
+    for (id, rate) in zip(node_id, flow_rate)
+        # Convention: edge points away from the flow boundary
```

Comment:
Here, and in the docs below you call it a convention, which suggests that either way would work. But looking at the function below, at best you get a "FlowBoundary intake not a Basin" error, and at worst we remove water from basins below 0 m3. So we should probably call it a requirement, and in #200 add validation checks for it.